doc/source/man.rst

```diff
@@ -250,54 +250,52 @@ As the network infrastructure changes, it might be necessary to delete some netw
 Facts
 -----
 
-The ``fact`` command is used to understand information that can be reported or to alter the contents of a report created from the ``rho scan`` command.
+Use the ``rho fact`` command to view information that can be reported in a scan or to alter the contents of a report that is created from the ``rho scan`` command.
 
 Listing Facts
 ~~~~~~~~~~~~~
 
-A list of facts that can be gathered during the scanning process can be obtained with the ``list`` command.
+To generate a list of facts that can be gathered during the discovery scanning process, use the ``rho fact list`` command.
```

Comment:
should we be consistent an say discovery and inspection scanning process?

Reply:
Change made.
I did a global search for "discovery" and added an "and inspection" in locations where it was missing.